Formula/libemf2svg.rb

```diff
@@ -0,0 +1,31 @@
+class Libemf2svg < Formula
+  desc "Microsoft (MS) EMF to SVG conversion library"
+  homepage "https://github.com/kakwa/libemf2svg"
+  url "https://github.com/kakwa/libemf2svg/archive/refs/tags/1.1.0.tar.gz"
+  sha256 "ad48d2de9d1f4172aca475d9220bbd152b7280f98642db561ee6688faf50cd1e"
+  license "GPL-2.0-only"
+
+  depends_on "argp-standalone" => :build
+  depends_on "cmake" => :build
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "libpng"
+
+  resource("testdata") do
```

Comment:
This stops our automation from suggesting updates with every release
```suggestion
  resource("homebrew-testdata") do
```